assets/js/modules/search-console/components/common/AnalyticsStats.js

```diff
@@ -100,6 +109,50 @@ export default function AnalyticsStats( props ) {
 	const analyticsModuleActive = useSelect( ( select ) =>
 		select( CORE_MODULES ).isModuleActive( moduleSlug )
 	);
+	const isGA4DashboardView = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).isGA4DashboardView()
+	);
+	const isGA4Connected = useSelect( ( select ) =>
```

Comment:
Hmm, technically you could just use `isGA4DashboardView` without explicitly checking for `isGA4Connected`, as `isGA4DashboardView()` returns false when GA4 is not connected...

Although, it could be better left as it is, to be a bit more explicit. What do you think?

Reply:
Interesting point, might indeed be worth considering, but agreed it's not something to introduce right here.